pkg/processor/constraints/constraint.go

```diff
@@ -0,0 +1,44 @@
+package constraints
+
+import (
+	"github.com/arttor/helmify/pkg/helmify"
+	corev1 "k8s.io/api/core/v1"
+)
+
+const topologyExpression = "\n{{- if .Values.topologySpreadConstraints }}\n" +
+	"      topologySpreadConstraints: {{- include \"tplvalues.render\" (dict \"value\" .Values.topologySpreadConstraints \"context\" $) | nindent 8 }}\n" +
+	"{{- end }}\n"
+
+const nodeSelectorExpression = "{{- if .Values.nodeSelector }}\n" +
+	"      nodeSelector: {{- include \"tplvalues.render\" ( dict \"value\" .Values.nodeSelector \"context\" $) | nindent 8 }}\n" +
+	"{{- end }}\n"
+
+const tolerationsExpression = "{{- if .Values.tolerations }}\n" +
+	"      tolerations: {{- include \"tplvalues.render\" (dict \"value\" .Values.tolerations \"context\" .) | nindent 8 }}\n" +
+	"{{- end }}\n"
+
+// ProcessSpecMap adds 'topologyConstraints' to the podSpec in specMap, if it doesn't
+// already has one defined.
+func ProcessSpecMap(specMap string, values *helmify.Values, podspec corev1.PodSpec) string {
+
+	(*values)["topologySpreadConstraints"] = podspec.TopologySpreadConstraints
```

Comment:
let's make some constants for the keys being used here (`topologySpreadConstraints`, `nodeSelector`, `tolerations`)